opengapps.xml

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
   <remote name="opengapps" fetch="https://github.com/opengapps/" />
```

Comment:
Why we do not change this?

Reply:
some of the repos are still hosted on github

Reply:
Hmm, OK :)

Then we should make `opengapps-gitlab`, as `gitlab` is just too generic.

Reply:
see here: https://github.com/opengapps/aosp_build

Reply:
done